modules/test/playwright/tests/export-import-web/main/instance.import.spec.ts

```diff
@@ -104,6 +105,107 @@ test('can export and import custom object entries at instance level', async ({
 	);
 });
 
+test('can import account restricted entry when account does and does not exist in enviroment', async ({
+	apiHelpers,
+	companyExportImportPage,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount();
+
+	apiHelpers.data.push({
+		id: account.id,
+		type: 'account',
+	});
+
+	const objectDefinition =
+		await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			status: {code: 0},
+		});
+
+	const objectRelationshipAPIClient = await apiHelpers.buildRestClient(
+		ObjectRelationshipAPI
+	);
+
+	const {body: objectRelationship} =
+		await objectRelationshipAPIClient.postObjectDefinitionByExternalReferenceCodeObjectRelationship(
+			'L_ACCOUNT',
+			{
+				label: {
+					en_US: 'objectRelationshipLabel' + getRandomInt(),
+				},
+				name: 'objectRelationshipName' + Math.floor(Math.random() * 99),
+				objectDefinitionExternalReferenceCode1: 'L_ACCOUNT',
+				objectDefinitionExternalReferenceCode2:
+					objectDefinition.externalReferenceCode,
+				type: 'oneToMany',
+			}
+		);
+
+	apiHelpers.data.push({
+		id: objectRelationship.id,
+		type: 'objectRelationship',
+	});
+
+	const objectDefinitionAPIClient =
+		await apiHelpers.buildRestClient(ObjectDefinitionAPI);
+
+	await objectDefinitionAPIClient.patchObjectDefinition(objectDefinition.id, {
+		accountEntryRestricted: true,
+		accountEntryRestrictedObjectFieldName: `r_${objectRelationship.name}_accountEntryId`,
```

Comment:
Could you declare `accountEntryId` before the  `patchObjectDefinition` and use it here.